.github/workflows/clean-rev-image.yml

```
@@ -0,0 +1,96 @@
+        
+        # Extract tags and count them
+        tag_count=$(echo $response | jq '.results | length')
+
+        # Set output based on the number of tags
+        echo "user_service_tag_count=$tag_count" >> $GITHUB_ENV
+        echo "user_service_tags=$response" >> $GITHUB_ENV
+
+    # Cleanup images for auth-service
+    - name: Cleanup auth-service Docker tags
+      id: cleanup_auth_service
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:3:1: THRESHOLD appears unused. Verify use (or export if used externally)

(shellcheck)